src/accountsdb/db.zig

```diff
@@ -189,7 +191,7 @@ pub const AccountsDB = struct {
     bank_hash_stats: RwMux(BankHashStatsMap) = RwMux(BankHashStatsMap).init(.{}),
 
     stats: AccountsDBStats,
-    logger: Logger,
+    logger: *Logger,
```

Comment:
Since Logger itself only contains a pointer, it should be passed by value.

Reply:
Done in [e6f3f13](https://github.com/Syndica/sig/pull/277/commits/e6f3f13a312f7f8e9ca90267d578a0c99971ee23)